invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/LinksTable.js

```diff
@@ -0,0 +1,74 @@
+// This file is part of InvenioRdmRecords
+// Copyright (C) 2022 CERN.
+//
+// Invenio RDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import PropTypes from "prop-types";
+import React, { useContext } from "react";
+import { i18next } from "@translations/invenio_administration/i18next";
+import CopyButton from "./CopyButton";
+import { Table, Header, Grid } from "semantic-ui-react";
+import AdminDetailsContext from "@js/invenio_administration/details/AdminDetailsContext.js";
+
+function LinksTable({ }) {
+
+  /**
+   * WIP: this method should remove metadataPrefix from the url once the attribute is listed
+   * as a dropdown.
+   *
+   * @param {string} url
+   * @returns {string} stripped url
+   */
+  const sanitizeLink = (url) => {
```

Comment:
This function should remove `metadataPrefix` from the url as this prefix will be rendered differently. Namely, it will be a dropdown / multi select .